kong/global.lua

```diff
@@ -201,6 +201,33 @@ function _GLOBAL.init_worker_events()
     return nil, err
   end
 
+  local PAYLOAD_MAX_LEN, PAYLOAD_TOO_BIG_ERR = 65535, "failed to publish event: payload too big"
+  local TRUNCATED_PREFIX = ", truncated payload: "
+  local DEFAULT_TRUNCATED_PAYLOAD = TRUNCATED_PREFIX .. "not a serialized object"
+  local LEN = (#PAYLOAD_TOO_BIG_ERR + #DEFAULT_TRUNCATED_PAYLOAD) * 2
```

Comment:
Could we rename `LEN` to a more meaningful name?

Reply:
Sure, a meaningful name would be better, however I don't have a good name for it, and I think it's simple enough to be explained by the code, so I just give it a simplest name. Sorry for making it confusing.